web-app/css/imosTheme.css

```diff
@@ -337,5 +337,14 @@ ul.x-tab-strip-top {
     border-radius: 6px;
 }
 
+.subsetPanelAccordionItem .x-tool:not(:first-of-type) {
+    margin: 3px 0;
+}
+
+.subsetPanelAccordionItem .x-tool:first-of-type {
+    margin-left: 6px;
+    margin-right: 0;
+}
+
```

Comment:
mmm... My thoughts are that themeing is useless as everyone starts with the imosTheme anyway.
This change affect prettiness only and as such why not keep it for IMOS